app/javascript/shared/components/useKeyboardShortcuts.jsx

```diff
@@ -31,19 +36,68 @@ function isFormField(element) {
  *   "ctrl+alt+KeyG": (e) => {
  *     e.preventDefault();
  *     alert("Control Alt G has been pressed");
+ *   },
+  *   "KeyG~KeyH": (e) => {
+ *     e.preventDefault();
+ *     alert("G has been pressed quickly followed by H");
  *   },
  *   "?": (e) => {
  *     setIsHelpVisible(true);
  *   }
  * }
  *
- * useKeyboardShortcuts(shortcuts, someElementOrWindowObject);
+ * useKeyboardShortcuts(shortcuts, someElementOrWindowObject, {timeout: 1500});
  *
  * @param {object} shortcuts List of keyboard shortcuts/event
  * @param {EventTarget} [eventTarget=window] An event target.
+ * @param {object} [options = {}] An object for extra options
  *
  */
-export function useKeyboardShortcuts(shortcuts, eventTarget = window) {
+export function useKeyboardShortcuts(shortcuts, eventTarget = window, options = { timeout: 500 }) {
```

Comment:
Refactor (blocking): The `timeout` property set to `500` is not necessary as the spread `defaultOptions` for props already handles that case. I always have maintenance on the brain. 😄

```suggestion  
export function useKeyboardShortcuts(shortcuts, eventTarget = window, options = {}) {
```